### ISSUE TITLE
is:issue is:open how to make a test??

### ISSUE THREAD

**Issue by zlpsophina** [lisjin/dcan-tensorflow#8]
hi，i have trained the model successful，then i want to  make a test by  a new image and create the contour and segment image or label consistent with this test image,so i want know weather just to run the "images=bbbc006.inputs(eval_data=FLAGS.eval_data)" and "c_fuse, s_fuse = bbbc006.inference(images, train=False)"under the pre-train model or not ,
i will be appreciate for your response.

**Comment by lisjin** [comment]
If I understand correctly, you would just need to run [`evaluate`](https://github.com/lisjin/dcan-tensorflow/blob/9e44ac1a1810655a0b0a5cb4b999654ce799b5fe/tf-dcan/bbbc006_eval.py#L98). You would also need to change the `FLAGS` globals at the top of the file to make sure it's reading inputs from the correct path.
